backend/src/controllers/__tests__/CMSLoginController.unit.test.ts

```diff
@@ -0,0 +1,109 @@
+import Server from "../../App";
+import supertest from "supertest";
+import { MongoClient, Db, Collection } from "mongodb";
+import { UNAUTHORISED_TEXT, BAD_REQUEST_TEXT } from "../../constants";
+
+/**
+ * This test utilises [supertest]{@link https://www.npmjs.com/package/supertest}
+ * and initialises the full server to perform the test per test.
+ *
+ * @author ale8k
+ */
+describe("CMSLoginController", () => {
+    let connection: MongoClient;
+    const collectionName = "cms-users";
+    let db: Db;
+    let collection: Collection;
+
+    beforeAll(async () => {
+        process.env.PORT = "3003";
+        process.env.DB_URL = global.__MONGO_URI__;
+        process.env.DB_NAME = global.__MONGO_DB_NAME__;
+        process.env.BCRYPT_SALT = "12";
+        process.env.JWT_KEY = "iamakeylol";
+        process.env.JWT_EXPIRY = "2 days";
+
+        connection = await MongoClient.connect((global.__MONGO_URI__), {
+            poolSize: 10,
+            useNewUrlParser: true,
+            useUnifiedTopology: true
+        });
+
+        db = await connection.db(global.__MONGO_DB_NAME__);
+        collection = db.collection(collectionName);
+        await collection.insert(
+            {
+                username: "alex",
+                password: "$2b$12$Cp/IwyOUsKiJZTANWklBB.k4mv07lIV1gSLT4FbtsOI.eoFi2qfTu"
+            }
+        );
+    });
+
+    afterAll(() => {
+        connection.close();
+    });
+
+    /**
+     * /CMS/AUTH/LOGIN
+     */
+    it("/cms/auth/login should login successfully, with 200 response", async (done) => {
+        await new Server({ path: ".env.testing" }).setupServer().then((app) => {
+            supertest(app).post("/cms/auth/login").send({ username: "alex", password: "p123" })
+                .expect(200)
+                .expect(response => {
+                    expect(response.body.successfulLogin).toBe(true);
+                }).end(done);
+        });
+    });
```

Comment:
I kind of loved the comment never seen that. 
anyway  `await` are mostly used for asynchronous behaviour and don' think its fine to use then like that even it's fine.
can you update it to something like 
```js
const json = await fetch()
const x = await json;
x.toMatch({})
```

Reply:
Yup, await isn't needed for these.  It's better off to not use it whatsoever when creating 'new Server'. Will change now.